Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -208,3 +208,22 @@ extension TopologyDescription.TopologyType {
         }
     }
 }
+
+public struct TestError: LocalizedError {
```

Comment:
this was being used across a few different test files and it seems general enough that it should just live here.